composer.json

```diff
@@ -10,44 +10,42 @@
     "spryker/kernel": "^3.0.0",
     "spryker/mail": "^4.0.0",
     "spryker/money": "^2.0.0",
-    "spryker/oms": "^6.0.0 || ^7.0.0",
-    "spryker/payment": "^3.0.0 || ^4.0.0",
+    "spryker/oms": "^7.0.0",
+    "spryker/payment": "^4.0.0",
     "spryker/propel-orm": "^1.0.0",
-    "spryker/sales": "^6.0.0 || ^7.0.0 || ^8.0.0",
+    "spryker/refund": "^5.0.0",
+    "spryker/sales": "^8.0.0",
     "spryker/session": "^3.0.0",
-    "spryker/step-engine": "^2.0.0 || ^3.0.0",
+    "spryker/step-engine": "^3.1.0",
     "spryker/symfony": "^3.0.0",
+    "spryker/transfer": "^3.0.0",
     "spryker/util-text": "^1.1.0",
     "spryker/zed-request": "^3.0.0"
   },
   "require-dev": {
     "spryker/code-sniffer": "*",
-    "spryker/country": "*",
-    "spryker/customer": "*",
     "spryker/propel": "*",
     "spryker/silex": "*",
     "spryker/testify": "*"
   },
-  "suggest": {
-    "spryker/sales": "If you want to use Oms command plugins."
+  "replace": {
+    "spryker/payolution": "*"
   },
   "autoload": {
     "psr-0": {
-      "Spryker": "src/",
-      "Function": "tests/"
+      "SprykerEco": "src/"
     }
   },
   "autoload-dev": {
     "psr-0": {
-      "Functional": "tests/",
-      "Unit": "tests/"
+      "SprykerEcoTest": "tests/"
     }
   },
   "minimum-stability": "dev",
   "prefer-stable": true,
   "extra": {
     "branch-alias": {
-      "dev-master": "3.0.x-dev"
+      "dev-master": "4.0.x-dev"
```

Comment:
Are you releasing 1.0 here? I think that would make sense, so you can here use 1.0....

Reply:
:) good idea — so far no info about versioning was given, so I just _versioned-up_